src/transformStateWithClones.js

```diff
@@ -7,7 +7,41 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const stateReport = [];
+  const currentState = { ...state };
+
+  for (const i of actions) {
+    if (i.type === 'clear') {
```

Comment:
Испольхуй switch..case
```suggestion
    switch (action.type) {
```